blocks/api/registration.js

```diff
@@ -190,7 +168,7 @@ export function unregisterBlockType( name ) {
  * @param {string} name Block name.
  */
 export function setUnknownTypeHandlerName( name ) {
-	unknownTypeHandlerName = name;
+	dispatch( 'core/blocks' ).setFallbackBlockType( name );
```

Comment:
Why do we call it the "fallback type" now, but the current API refers to it as the "unknown type" ?

Reply:
Also, the choice of referring to it by "name" was intentional; type is the object, we're setting, storing, and retrieving it as a name.

Reply:
I feel like fallback is better name for consistency with the defaultBlockType name. 

so instead of `unknownTypeHandlerName` and `defaultBlockTypeName`
we would have `fallbackBlockTypeName` and `defaultBlockTypeName`.

I can update to add the "name" suffix to the actions as well, it's already used in the selectors.

Reply:
But then should we want to update the block API to expose `setFallbackBlockTypeName` as well? Also, is it `xBlockTypeName` or `xBlockName` ?

Reply:
Sure, I can use the latter. Though, I don't want to change the API for now, Let's avoid deprecations until we figure out server-side awareness.